src/main/java/uk/gov/hmcts/darts/common/entity/TranscriptionDocumentEntity.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.darts.common.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+
+import java.time.OffsetDateTime;
+
+@Entity
+@Table(name = "transcription_document")
+@Getter
+@Setter
+@EqualsAndHashCode(callSuper = false)
+public class TranscriptionDocumentEntity {
+    @Id
+    @Column(name = "trd_id")
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "trd_gen")
+    @SequenceGenerator(name = "trd_gen", sequenceName = "trd_seq", allocationSize = 1)
+    private Integer id;
+
+
+    @Column(name = "tra_id", nullable = false)
+    private Integer transcriptionId;
+
+    @Column(name = "file_name", nullable = false)
+    private String fileName;
+
```

Comment:
It looks like we're missing a column for fileType?
`file_type                   CHARACTER VARYING             NOT NULL`

Reply:
ACCEPT